scripts/pull_github_pr.sh

```diff
@@ -11,10 +11,43 @@ set -e
 
 gh_hosts=~/.config/gh/hosts.yml
 jenkins_url="https://jenkins.scylladb.com"
-FORCE=$2
 ORANGE='\033[0;33m'
 NC='\033[0m'
 
+if [ $# -lt 1 ]; then
+  echo Please provide a github pull request number
+fi
+
+PR_NUM=$1
```

Comment:
It is strange to have to pass the flags after the PR number. Can we extract it in the loop below (write it in the else case).

Reply:
Fixed it, now you can add the flags in any order